libs/blocks/aside/aside.js

```diff
@@ -24,23 +28,59 @@ function decorateLayout(el) {
   return foreground;
 }
 
-function decorateContent(el, isInline) {
+function decorateContent(el, type, size) {
   if (!el) return;
-  const text = el.querySelector('h1, h2, h3, h4, h5, h6')?.closest('div');
+  const text = el.querySelector('h1, h2, h3, h4, h5, h6, a')?.closest('div');
+  const picture = text?.querySelector('picture');
+  const iconArea = picture ? (picture.closest('p') || createTag('p', null, picture)) : null;
   text?.classList.add('text');
+  if (text && !text.querySelector('p')) {
+    const buttons = text.querySelectorAll('em a, strong a');
+    let btnWrap = null;
+    if (buttons[0] && !buttons[0].closest('p')) {
+      btnWrap = document.createElement('p');
+      buttons.forEach((button) => btnWrap.append(button.parentElement));
```

Comment:
nit: best practice is to call append once with multiple els
```suggestion
      buttons.forEach((button) => btnWrap.append(button.parentElement));
      btnWrap.append(...buttons.map(b => b.parentElement));
```